src/env/hedged/util.rs

```diff
@@ -0,0 +1,9 @@
+use std::path::{Path, PathBuf};
+
+pub fn replace_path(path: &Path, from: &Path, to: &Path) -> PathBuf {
```

Comment:
The fn can be renamed as replace_prefix? Otherwise it's not obvious that parameter from must be prefix of path.